main.go

```diff
@@ -59,7 +59,7 @@ func serve(revision string, cfg *config.Configuration) error {
 	currencyConverterTickerTask := task.NewTickerTask(fetchingInterval, currencyConverter)
 	currencyConverterTickerTask.Start()
 
-	r, err := router.New(cfg, currencyConverter)
+	r, err := router.New(cfg, currencyConverter, revision)
```

Comment:
Cool, added the new variable but would need help modifying the Dockerfile because I can't find where the Rev string is actually populated... if it's actually not set I can add both Version and Rev to the build script. 
I would also need help describing how it's populated in the comment above the variable since i'm not sure if the Rev part is still true.

Reply:
It appears we are not actually setting the the Rev during the build process for the Docker image. We also used to build a binary and did specify it there, but we removed that artifact as no one was using it. 

> if it's actually not set I can add both Version and Rev to the build script.

Yes, please do. Bonus points if you test the Docker build process locally to ensure the variables are properly set. You would make the change in Dockerfile on line 23 `RUN go build -mod=vendor .`

Reply:
Added both flags to the Dockerfile. Tested locally and in Docker that both are correctly set.

Reply:
> Tested locally and in Docker that both are correctly set.

Yay! That's fantastic. Thank you.